themes/bootstrap3/less/components/search.less

```diff
@@ -209,6 +209,11 @@ header .container.navbar { margin-bottom: 0; }
   display: none;
 }
 
+.savedLists__ul {
```

Comment:
Is this style change directly related to other changes, or something that got tweaked along the way?